MailSenderTask2/src/main/java/com/demo/controller/MailSenderController.java

```diff
@@ -0,0 +1,40 @@
+package com.demo.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+
+import com.demo.model.MailSender;
+import com.demo.service.MailSenderService;
+
+@RestController
+public class MailSenderController {
+
+	@Autowired
+	private MailSenderService mailSenderService;
+	
+	@RequestMapping("/list")
```

Comment:
URL is a name. URL should be a noun. Not a verb! Use request method to explain what to do. Example: POST /emails - means that you post(or add to simplify) something to collection of emails. Here is the good explanation of how to create good urls https://www.restapitutorial.com/lessons/restfulresourcenaming.html